R/pkg/inst/tests/testthat/test_context.R

```diff
@@ -24,10 +24,18 @@ test_that("Check masked functions", {
   func <- lapply(masked, function(x) { capture.output(showMethods(x))[[1]] })
   funcSparkROrEmpty <- grepl("\\(package SparkR\\)$|^$", func)
   maskedBySparkR <- masked[funcSparkROrEmpty]
-  namesOfMasked <- c("describe", "cov", "filter", "lag", "na.omit", "predict", "sd", "var",
-                     "colnames", "colnames<-", "intersect", "rank", "rbind", "sample", "subset",
-                     "summary", "transform", "drop", "window", "as.data.frame",
-                     "endsWith", "startsWith")
+  if (as.numeric(R.version$major) == 3 && as.numeric(R.version$minor) > 2) {
```

Comment:
can we just append to the list ? i.e. create `namesOfMasked` outside this and if the version number is greater than 2 we can do 
`namesOfMasked <- c("endsWith", "startsWith", namesOfMasked)`